src/Workspaces/MSBuildTest/OpenProjectsTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.MSBuild.UnitTests
+{
+    public class OpenProjectsTests : MSBuildWorkspaceTestBase
```

Comment:
```suggestion
    public class NewlyCreatedProjectsFromDotNetNew : MSBuildWorkspaceTestBase
```

Just to make it really clear what this is.

Reply:
Thanks for taking the naming burden off of me.